src/plugins/soundChanger.tsx

```diff
@@ -0,0 +1,155 @@
+                                                    if (s.name === sound.name) {
+                                                        s.new_link = e.nativeEvent.target.value;
+                                                        setSoundsChanged(_ => { save(soundsChanged); return soundsChanged; });
+                                                        break;
+                                                    }
+                                                }
+                                            }}
+                                            value={soundsChanged.find(s => s.name === sound.name)?.new_link}
+                                        />
+                                    </td>
+                                    <td
+                                        style={{ transform: "translateY(-40%)", width: "5%" }}
+                                        onClick={() => {
+                                            const newSounds = soundsChanged.filter(s => s.name !== sound.name);
+                                            setSoundsChanged(_ => { save(newSounds); return newSounds; });
+                                        }}>
+                                        <span style={{ cursor: "pointer" }}>
+                                            <DeleteIcon width="24" height="24" color="var(--status-danger)" />
+                                        </span>
+                                    </td>
+                                </tr>
+                            );
+                        })
+                    }
+                    {!soundsChanged.length &&
+                        <Text style={{
+                            transform: "translateX(50%)",
+                            marginTop: "10%",
+                            marginBottom: "25%"
+                        }} variant="text-sm/normal">No sounds have been changed.</Text>
+                    }
+                </tbody>
+            </table>
+
+            <div style={{ marginBottom: "5%" }}>
+                <Select
+                    options={
+                        Object.keys(sounds)
+                            .filter(sound => soundsChanged.find(a => a.name === sound) === undefined)
+                            .map(sound => ({ label: sound.replace("./", ""), value: sound }))
+                    }
+                    placeholder="Change a sound"
+                    maxVisibleItems={5}
+                    closeOnSelect={true}
+                    isSelected={() => false}
+                    serialize={a => String(a)}
+                    select={(sound: string) => {
+                        const filename = `${wreq.m[sounds[sound]]}`.match(/"(.*?\.mp3)"/)!![1];
+                        const newSounds = soundsChanged.concat([{ name: sound, original_link: filename, new_link: "" }]);
+                        setSoundsChanged(_ => { save(newSounds); return newSounds; });
```

Comment:
should prob just do
```suggestion
                        setSoundsChanged(newSounds);
```

for this one, cause `save(...)` would be pointless since it filters out any empty sounds, and it is bound to be empty at this point in time

Reply:
I will push a commit for this btw, its just that I noticed it whilst making my review